src/ui/actions/timeline.ts

```diff
@@ -184,12 +200,22 @@ function updatePausePointParams({
   point,
   time,
   hasFrames,
+  focusRegion,
 }: {
   point: ExecutionPoint;
   time: number;
   hasFrames: boolean;
+  focusRegion: FocusRegion | null;
 }) {
-  const params = { point, time: `${time}`, hasFrames: `${hasFrames}` };
+  const params: { point: string; time: string; hasFrames: string; focusRegion?: string } = {
+    point,
+    time: `${time}`,
+    hasFrames: `${hasFrames}`,
+    focusRegion: undefined,
```

Comment:
I don't think it's the time to sound alarms yet, but we probably should take a look at how/if it's worth simplifying this when we get to parameter #5

Reply:
🤷‍♂️